### DISCUSSION TITLE
議事メモ

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#583] [General]
議事メモ

**Comment by Ayato-kosaka** [comment]
# 2023/01/24 Spelieve 検討会
## マーケディング（野口）
### 次回までに確認してほしいこと
* web や Google Play を想定した図にしてほしい
* SNS としての循環を意識できるフレームワークにしたい

**Reply by Ayato-kosaka** [reply]
SNS としての特徴を活かしたい

**Reply by Ayato-kosaka** [reply]
人が少ないと、使えないから、人を増やすのがまず大事なのではないか

**Reply by Ayato-kosaka** [reply]
なので、初めのうちは一人でも使えるを重視したアプリを作るべきではないか。

**Reply by Ayato-kosaka** [reply]
Ver 4 を Ver 3 より先にやるのも有りかも

**Reply by Ayato-kosaka** [reply]
Ver4 の場所斡旋もKPI分解したい

**Reply by Ayato-kosaka** [reply]
青は、「iOS日本語」「iOS英語」「GooglePlay日本語」「GooglePlay英語」「Web日本語」「Web英語」の6チャネルにしたい

**Comment by Ayato-kosaka** [comment]
# 2023/02/02 Spelieve 検討会

**Reply by Ayato-kosaka** [reply]
KGI から立てる vs 緑から立てる
右から立てると現実的、KGIから立てると理想的になる。
=> 目標をから数値を出し方が良いアイデアが出そうなので、KGI から立てる

シミュレーションをする目的
作戦の検証をすること
実値と作戦の距離を測ること

作戦を検証するためにすること
KPIの目安値が無いと、作戦を評価することが難しい

作戦立て-> シミュレーションのサイクルを早める
幅広く作戦を検討し、即シミュレーションをやる

**Comment by Takapy1** [comment]
# 2023/02/09 Spelieve 検討会
## 議題
- [x] 作業結果、問題点共有
- [x] Kanban調整
- [x] コスト見積もり
- [x] 広告設置場所
- [x] Visionなどの再検討
- [x] マーケティングプラン(KGI決定)

## 検討結果
### 広告設置場所


### Vision 検討

### 次回までの確認事項
*Admobブランチ←Ansense
* PGDATAにCCO007追加

**Reply by Ayato-kosaka** [reply]
入れる箇所 -> ページの一番上と一番下（ヘッダーとフッターは見える）

FuncID | FuncName | Memo
-- | -- | --
CPA001 | HelloSpelieve |  いれない
IPA001 | ItineraryEdit |  入れる
IPA003 | EditPlan |  入れる
IPA004 | ItineraryPreview |  入れない
PPA001 | Places | 入れない
PPA002 | Place | 入れる

**Reply by Ayato-kosaka** [reply]
Active User | Instaの時間軸 | Spelieveの時間軸 | 売上 | Memo
-- | -- | -- | -- | --
0 | 2010/10/06 | 2023/01/05 |   | リリース
1,000,000 | 2010/12/01 | 2024/01/01 |   |  
5,000,000 | 2011/06/01 | 2024/07/22 |   |  
30,000,000 | 2012/04/01 | 2028/12/01 | 81,000,000,000 | Facebook 買収

**Comment by Ayato-kosaka** [comment]
# 2023/02/09 Spelieve 検討会
## 議題
* 変更管理運用を始めることにした

**Reply by Takapy1** [reply]
@Ayato-kosaka 
話し合って決めたことを、しっかり記録として残すってことかな？

**Reply by Ayato-kosaka** [reply]
@Takapy1 
月曜説明するつもり〜
リリース後のリソースのERD変更等は、リリース時に移行作業しないといけないから、備忘で残そうってこと！
今の所 ERD 変更しか思いついてないからそこまで木にしなくて大丈夫〜

例えば、カラム名変更したら、本番環境のデータも一括変更しないと元のデータが使えなくなる

**Comment by Takapy1** [comment]
# 2023/02/16 Spelieve 検討会
共有したいサイト
https://applion.jp/ipad/word/%E6%97%85%E3%81%AE%E3%81%97%E3%81%8A%E3%82%8AFree/

**Comment by Ayato-kosaka** [comment]
# 2023/03/16 Spelieve 検討会

**Reply by Ayato-kosaka** [reply]
** Story **
こんな課題を解決するための事業=需要=事業価値
その事業計画はしっかり組めている=事業計画
その事業は収益も見込めている=ビジネスモデル
こんな現状で対応している=企業価値
だから出資してほしい

**Reply by Ayato-kosaka** [reply]
** angel 探し **
1~2人にチャットを送ってみて温度感を測る

**Comment by Takapy1** [comment]
@Ayato-kosaka 
# 3/20 

# 議題
- 株式会社設立せずに、資金調達が不可能であることから、エンジェルからの調達の方法で継続すべきか

## 株式会社設立せずに、資金調達が不可能であることから、エンジェルからの調達の方法で継続すべきか
設立せずに、資金調達する他の方法→**クラファン**

### 検討結果
エンジェルでの調達を継続

### 理由
- 計画、資料を考えたときに、クラファンだと質の高いものにならない
理由：計画、資料を評価するのが自分たち、プロのフィードバックを受けられないから

- エンジェルの方が、プロのフィードバックを受けれ、いい計画になるから。

### 調達できそうならどうするのか?
→金額による。
こさが会社を辞めれる金額の調達ができた場合：株式会社を設立し、出資を受ける
少額or資金調達が受けれない場合：フィードバックをもらった資料を使って、クラファンなどの方法を移行

**Comment by Ayato-kosaka** [comment]
# 2023/03/23 Spelieve 検討会

**Reply by Ayato-kosaka** [reply]
https://docs.google.com/presentation/d/1NWF2jpO0IaeUGWCoiXM9b-yuBc3_zQUC7SRXXiJMfBM/edit#slide=id.g2232a3810b9_0_174

**Reply by Ayato-kosaka** [reply]
**事業価値の根拠付けはどうする？？**
根拠付をベンチャー事業で定量的に主張するのは難しい。
事業価値、ビジネスモデルで「これならきっと、うまくいく」というところまでを確実に出せれば良さそう

**今後の計画どうするか**
9ページまでを固める作業に入る。=>磨き上げに入る
根拠付けの固めは、終わりがないので、メインスライドに入れない

**Reply by Ayato-kosaka** [reply]
LinkedIn 使ってみた。
フォロワーをフォローしてる人までしかフォローできない。
DMするにはフォローされないといけない。

AngelList
会社名が無いと登録できない。

Angel 探しが課題有り。ウェイトを少し上げて探してみる。

**Comment by Takapy1** [comment]
# 2023/04/3 Spelieve 検討会

## 議題
- エンジェル探しに少し限界がきている

## エンジェル探しに関して
## 問題
様々な方法でエンジェルとのコンタクトを試みているが、どれもうまく進まず、アイデアの限界＋ 見つけるまでの期限がやばいから解決したい

## 現状
- Twitter: DM送りまくり作戦→送れない、投稿にコメント：反応なし
- LinkedIn：フォローワーが広がらない、遅い
- その他ぷらっとフォーム系：USからのアクセスしかできない(理由不明)or会社の登録が必要なため、できなかった

## 決定事項
日本のエンジェルプラットフォームを使って、エンジェル探しをする
ピッチ発表を優先
デメリットとして
ビジネスプランのクオリティ、調達額下がりそうだけどしょうがない

**Comment by Ayato-kosaka** [comment]
2023/10/14

**Reply by Ayato-kosaka** [reply]
話したいこと忘れないようにアジェンダメモっておくスレッド（@Takapy1 ）

**Reply by Ayato-kosaka** [reply]
・ホーム画面にデモ入れたい話
・次のタスクの話
　　　・タスク整理は完了して大事そうなやつの色付けはした
　　　・その他アイデアは飲み会で話したほうが良いかな？
